obs2ioda-v2/src/define_mod.f90

```diff
@@ -98,7 +98,7 @@ module define_mod
 
 character(len=nstring), dimension(ninst_geo) :: geoinst_list = &
    (/                     &
-      'ahi_himawari8   '  &
+      'ahi_himawari9   '  &                              ! MRI -- changed from ahi_himawari8 to ahi_himawari9
```

Comment:
You should add himarwari9 instead of replacing 8 with 9. Code needs to work with both h8 and h9.